required/latex-lab/latex-lab-table.dtx

```diff
@@ -546,20 +546,34 @@
 % \enquote{vmode} tables like longtable.
 %
 % \begin{plugdecl}{Table}
+%  Inside a TD or TR Part is not allowed as child. For structures that restore
+%  paragraph settings we therefore need a special plug that adjust the settings.
+%  Currently we set the para main tag to Div. This leads to double Div-structures
+%  but flattening the paragraph doesn't work, it errors if there is a list inside. 
+%    \begin{macrocode}
+\NewSocketPlug{tagsupport/para/restore}{Table}
```

Comment:
maybe this socket is better called "/adjust" if it is changing the para handling?

Reply:
well the general plan is to put this inside \@parboxrestore  at some time, so I think the name fits.

Reply:
well if it goes into parboxrestore then it isn't "table" specific in which case the plug name is wrong. If, however, we anticipate different plugs, adjusting  aspects of the paratagging then it still seems to to me that "adjust" is correct even if it is called in parbox restore